packages/web-react/src/components/UNSTABLE_Truncate/stories/UNSTABLE_Truncate.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import ReadMe from '../README.md';
+import { UNSTABLE_Truncate } from '..';
+
+const meta: Meta<typeof UNSTABLE_Truncate> = {
+  title: 'Experimental/UNSTABLE_Truncate',
+  component: UNSTABLE_Truncate,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'text',
+    },
+    lines: {
+      control: 'number',
+    },
+  },
+  args: {
+    children:
```

Comment:
please add the `lines` here as well with the default value set

Reply:
if it isn't set here, it still appears in the table, but you have click once more to edit it.
Current state:
<img width="1125" alt="image" src="https://github.com/user-attachments/assets/b4994026-65a9-446a-a8d0-05ffeb5a5ed1">
If the `lines` object is already set here:
<img width="1197" alt="image" src="https://github.com/user-attachments/assets/760e9059-b27f-4af3-98c8-bd71ade8fa79">